modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -186,6 +183,42 @@ include::partial$include-reauthentication-conditions.adoc[]
 
 Connected apps and clients are exempt from reauthentication. If you have a script that makes an API call to change your email address, it might need to accommodate reauthentication.
 
+=== Verify Your Email Address
+
+If you have not verified your email address that you use to log in to Anypoint Platform, a prompt appears to verify your email when you log in.
+You can verify your email using the login prompt, or you can verify your email in your user profile.
+If you are an SSO user, you cannot verify your email address through Anypoint Platform.
+
+include::partial$include-nav-steps-to-profile.adoc[]
+
+. Click the *Contact Info* tab.
+. In the *Email* section, click *Verify Email*. +
+A window appears, prompting you to enter a verification code that you received at your Anypoint Platform account's email address.
++
+. Enter the verification code.
+. Click *Confirm*. +
+Your email address is verified. 
+
+== Change Your Account Email Address
+
+You can change the email address that you use to log in to your Anypoint Platform account.
+
+[NOTE]
+SSO users cannot change their account email address through Anypoint Platform.
+
+include::partial$include-nav-steps-to-profile.adoc[]
+
+. Click the *Contact Info* tab.
+. In the *Email* row, click the pencil icon.
+. Enter your new email address.
+. Click *Save Changes*.
+. If you have been logged in to Anypoint Platform for more than 30 minutes, reenter your password to reauthenticate. +
+A window appears, prompting you to enter a verification code that you received at the new email address you entered.
++
+. Enter the verification code.
+. Click *Confirm*. +
+The email address you entered is now the email address you use to log in to Anypoint Platform.
```

Comment:
This isn't technically true as they use the username to login

Reply:
```suggestion
The email address you entered is now your primary email address for Anypoint Platform.
```